Sources/Filters/General/ContourLoopExtraction/index.js

```diff
@@ -0,0 +1,156 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+
+const Dir = {
+  Forward: 1,
+  Backward: -1,
+};
+
+const visited = new Set();
+
+function vtkContourLoopExtractionCtor(publicAPI, model) {
```

Comment:
Let's rename this to `vtkContourLoopExtraction` for consistency with the rest of vtk.js.